azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -62,12 +62,14 @@
 CPUAccounting=yes
 """
 _AGENT_DROP_IN_FILE_CPU_QUOTA = "12-CPUQuota.conf"
-_AGENT_DROP_IN_FILE_CPU_QUOTA_CONTENTS = """
+_AGENT_DROP_IN_FILE_CPU_QUOTA_CONTENTS_FORMAT = """
 # This drop-in unit file was created by the Azure VM Agent.
 # Do not edit.
 [Service]
-CPUQuota=5%
+CPUQuota={0}%
 """
+_AGENT_CPU_QUOTA = 5
```

Comment:
yes, before shipping. for testing i want to try 5%

Reply:
should have mentioned that i have a task to add these settings to the conf file. so eventually this will come from the conf file and the tests can change it as they need

Reply:
Ahh ok fair enough